internals/cli/cmd_help.go

```diff
@@ -101,11 +102,11 @@ type manfixer struct {
 func (w *manfixer) Write(buf []byte) (int, error) {
 	if !w.done {
 		w.done = true
-		if bytes.HasPrefix(buf, []byte(".TH pebble 1 ")) {
+		if bytes.HasPrefix(buf, []byte(fmt.Sprintf(".TH %s 1 ", cmd.Personality.ProgramName))) {
```

Comment:
It's somewhat subjective, but here's a case where I personally think the `Sprintf` is more readable than string concat, because the big long `cmd.Personality.ProgramName` variable name gets in the way of things. It seems more clear with the printf template string where the fiddly spaces are.

```go
fmt.Sprintf(".TH %s 1 ", cmd.Personality.ProgramName)
// vs
".TH " + cmd.Personality.ProgramName + " 1 "
```